app/models/container_image.rb

```diff
@@ -112,5 +112,9 @@ def disconnect_inv
     save
   end
 
+  def openshift_image?
+    command.present?
```

Comment:
We are only collecting this field from Openshift images and I expect all of them to have it. In case not all images have `command` and we can't find another field that answers these criteria then we will need to add something to mark it.

BTW this is currently will cause an error message when ManageIQ attempts to annotate an image that doesn't exists but this won't result in a negative user experience so maybe we also want to leave it as is.